### PR TITLE
Test user cannot put file in location without permissions

### PR DESCRIPTION
**Pull request by lucafavatellawork**
Test coverage increase as preparation for https://github.com/opensparkl/sse_core/issues/179

TODO:
- [ ] Once CI on master of this repo fixed, merge master branch into this repo (shall be possible to click in Web UI "update branch" or similar).

### CODE REVIEW

**Comment by lucafavatellawork** [comment]
I did not find a similar test for sse_core so I propose to add it here.

**Comment by lucafavatellawork** [comment]
Please notice that CI fails on master for https://github.com/opensparkl/sse_cli/issues/96 - see unrelated PRs for that.

File: test/bin/activate [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# This file must be used with "source bin/activate" *from bash*
```
Comment (miklosduma) [inline test/bin/activate:1]: I don't think you wanted to add in this file :)

File: test/bin/activate.csh [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# This file must be used with "source bin/activate.csh" *from csh*.
+# You cannot run it directly.
+# Created by Davide Di Blasi <davidedb@gmail.com>.
+# Ported to Python 3.3 venv by Andrew Svetlov <andrew.svetlov@gmail.com>
```
Comment (miklosduma) [inline test/bin/activate.csh:4]: This one neither

File: test/bin/activate.fish [hunk 3]
```diff
@@ -0,0 +1,75 @@
+# This file must be used with ". bin/activate.fish" *from fish* (http://fishshell.org)
+# you cannot run it directly
+
```
Comment (miklosduma) [inline test/bin/activate.fish:3]: And so on... maybe add `test/bin` to gitignore instead?

**Pull request merged**